exonum/src/node/events.rs

```diff
@@ -77,6 +77,10 @@ impl NodeHandler {
                 self.connect(&address);
             }
             ExternalMessage::Enable(value) => {
+                if !self.is_validator {
+                    error!("Trying to enable auditor node");
```

Comment:
Error message is not clear for user.
"Trying to enable consensus, but the current node is auditor and can not affect consensus process."